packages/react-components/react-search/library/src/components/SearchBox/useSearchBox.tsx

```diff
@@ -66,6 +66,8 @@ export const useSearchBox_unstable = (props: SearchBoxProps, ref: React.Ref<HTML
     const newValue = '';
     setInternalValue(newValue);
     props.onChange?.(event, { value: newValue });
+
+    searchBoxRef.current?.focus();
```

Comment:
I can't figure out why `searchBoxRef` was created & defined before this PR, but it looks good now! 😅

Reply:
yeah, probably an artifact from when I was clueless working on my first component haha